### ISSUE TITLE
Draw clear boundaries between proper CLDF and Dictionaria add-ons

### ISSUE THREAD

**Issue by xrotwang** [dictionaria/cldf2lift#1]
E.g. here https://github.com/dictionaria/cldf2lift/blob/5f623a4ba25ca9ff542c8e72863052117ae6eb27/src/cldf2lift/__init__.py#L7 we should use [`pycldf.Dataset.iter_rows`](https://github.com/cldf/pycldf/blob/d8ad5ad90b0ef1611ca7e2575016e3c485a96c44/src/pycldf/dataset.py#L725), to map the ontology term `description` to the local column name.
Things like `alt_translation2` are only Dictionaria conventions - which should be made clear.

**Comment by xrotwang** [comment]
Overall, this looks like a good test case for the CLDF [Dictionary module](https://github.com/cldf/cldf/tree/master/modules/Dictionary) and whether the spec is complete enough to allow for a useful conversion to LIFT.

**Comment by johenglisch** [comment]
Hm, aside from cases with multiple meta languages (`alt_translation[12]`) there is also the slight problem, that the entire way how we link examples and senses is a Dictionaria extension; i.e. there is no `senseReference` property in the standard that `Sense_IDs` could refer to.

One could maybe turn things around und have the senses refer into the example table instead (using `exampleReference`).  Technically the direction shouldn't matter, since it's _n→n_ either way.  Downside is that changing Dictionaria's database design would need a backwards-incompatible re-release of each and every dataset involved, which is… not ideal… (<_<)"

**Comment by xrotwang** [comment]
Yes, I think for the time being, I wouldn't change things and accept that even the core functionality is dictionaria-specific. Next step could be making the column names configurable via cli options, and then see whether there are actually use cases / users.
